go/src/meguca/server/init.go

```diff
@@ -114,7 +119,13 @@ func Start() error {
 	default:
 		return err
 	}
+
 	setConfigDefaults(&conf)
+	err = checkGeoLite()
```

Comment:
Also, I'd integrate this as part of loading the database in the geo-ip module. Will simplify the public interface.